.swm/doc-templates/a-flow-in-the-code-pqkt5.sw.md

```diff
@@ -0,0 +1,26 @@
+---
+id: pqkt5
+name: A flow in th ecode
+file_version: 1.0.2
+app_version: 0.7.1-1
+---
+
+## Introduction
+This doc describes the {{SUBJECT}} flow in our system. We will follow its implementation across the various locations so you can understand how the different parts create the full picture.
+
+## Following the flow
+
+<br/>
+<!-- TEMPLATE-swimm-snippet-placeholder -->
+Select the first step where the flow begins - e.g., the entry point, or an API call.
```

Comment:
Shouldn't this also be {{Select the first step where the flow begins - e.g., the entry point, or an API call.}}

Reply:
@nadav-swimm this is within the snippet placeholder - so it appears within the placeholder.. I think we shouldn't use {{ for these...